src/rpc/blockchain.cpp

```diff
@@ -719,9 +724,14 @@ UniValue getblock(const UniValue& params, bool fHelp)
             throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid block height parameter");
         }
 
+        if(nHeight < 0) {
```

Comment:
```suggestion
        if (nHeight < 0) {
```

Reply:
Sorry about that ... again. I changed it in the context of https://github.com/zcash/zcash/commit/3fb8af512c464140d6ca67c814313bd72bb17780